app/test_main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+from typing import Callable
+from unittest import mock
+from app.main import can_access_google_page
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    "url,valid_url,connection,result",
+    [
+        ("https://www.google.com.ua", True, True, "Accessible"),
+        ("https://www.google.com.ua", False, True, "Not accessible"),
+        ("https://www.google.com.ua", True, False, "Not accessible"),
+        ("https://www.google.com.ua", False, False, "Not accessible")
```

Comment:
It would be better to remove url from here, and paste it once in can_access_google_page("https://www.google.com.ua")

Reply:
@WDemidenko is right, make ur code DRY